style.css

```diff
@@ -0,0 +1,156 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* navbar section styling */
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 12px;
+  left: 0%;
+  right: 0%;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  position: absolute;
+  flex-wrap: wrap;
+  margin-right: 100px;
+}
+
+.navbar a {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 10px;
+  font-size: 19px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 10px;
+  text-decoration: none;
+}
+
+.navbar nav {
+  align-self: flex-end;
+}
+
+.navbar img {
+  display: flex;
+  right: 27.67%;
+  left: 16%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid #fff;
+  margin-bottom: 3px;
+  flex-grow: 0;
+  order: 0;
+  flex-direction: row;
+}
+
+#bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-self: flex-end;
+  margin: 2px 0 2px 10px;
+}
+
+#bar img {
+  display: flex;
+  right: 27%;
+  left: 16.67%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid white;
+  margin-bottom: 3px;
+}
+
+/* headline section styling */
+.headline {
+  overflow: hidden;
+  background-image: url(images/Group.png);
+  background-position-y: 125%;
+  background-color: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+  height: auto;
+  width: 100%;
+  padding: 160px 25px;
+  gap: 15px;
+  background-position: right bottom;
+  background-size: 145px;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.h1 {
+  color: #ff6b00;
+  width: 85%;
+  height: 5px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 210;
+  font-size: 44px;
+  line-height: 65px;
+  letter-spacing: 0.35px;
+  flex: none;
+}
```

Comment:
- [ ] We are very close to completing the project, let's be patient and keep the momentum. I have noticed that your `title` overlaps with the `sub-title`. Kindly ensure that you fix this so that your portfolio is professional and user-friendly ☺️.
Hint: You may consider removing the heights from `.h1`, and `.h4`. In addition, you may remove or reduce the margin-top of `h4`.

Your design  😎 
![image](https://user-images.githubusercontent.com/99394374/227151093-b0601eda-506d-4c08-b21f-dd72aa58b662.png)

Reply:
Thank you for helping me out. This is greatly appreciated. I have made the changes in the suggested area. Thanks again.